interface/main/finder/dynamic_finder.php

```diff
@@ -337,48 +341,41 @@ function persistCriteria(el, e) {
-    <div id="container_div" class="<?php echo attr($oemr_ui->oeContainer()); ?>">
+<body>
+    <div id="container_div" class="<?php echo attr($oemr_ui->oeContainer()); ?> mt-3">
          <div class="row">
-            <div class="col-sm-12">
-                <div class="page-header">
-                    <?php echo $oemr_ui->pageHeading() . "\r\n"; ?>
-                </div>
+            <div class="col-12">
+                <?php echo $oemr_ui->pageHeading() . "\r\n"; ?>
             </div>
-        </div>
-        <div class="row">
-            <div class="col-sm-12">
+            <div class="col-12">
                 <?php if (AclMain::aclCheckCore('patients', 'demo', '', array('write','addonly'))) { ?>
-                    <button id="create_patient_btn1" class="btn btn-secondary btn-add" onclick="top.restoreSession();top.RTop.location = '<?php echo $web_root ?>/interface/new/new.php'"><?php echo xlt('Add New Patient'); ?></button>
+                    <button id="create_patient_btn1" class="btn btn-primary btn-add" onclick="top.restoreSession();top.RTop.location = '<?php echo $web_root ?>/interface/new/new.php'"><?php echo xlt('Add New Patient'); ?></button>
                 <?php } ?>
             </div>
-            </div>
-        <br />
-        <div class="row">
-            <div class="col-sm-12">
+            <div class="col-12 jumbotron mt-3 p-4">
                 <div id="dynamic"><!-- TBD: id seems unused, is this div required? -->
                     <!-- Class "display" is defined in demo_table.css -->
-                    <table cellpadding="0" cellspacing="0" class="border-0 display" id="pt_table">
-                        <thead>
-                            <tr id="advanced_search" class="hideaway"  style="display: none;">
-                                <?php echo $header0; ?>
-                            </tr>
-                            <tr class="head">
-                                <?php echo $header; ?>
-                            </tr>
-                        </thead>
-                        <tbody>
-                            <tr>
-                                <!-- Class "dataTables_empty" is defined in jquery.dataTables.css -->
-                                <td class="dataTables_empty" colspan="<?php echo attr($colcount); ?>">...</td>
-                            </tr>
-                        </tbody>
-                    </table>
+                    <div class="table-responsive">
+                        <table class="table table-hover" cellpadding="0" cellspacing="0" class="border-0 display" id="pt_table">
+                            <thead>
+                                <tr id="advanced_search" class="hideaway"  style="display: none;">
+                                    <?php echo $header0; ?>
+                                </tr>
+                                <tr class="head">
+                                    <?php echo $header; ?>
+                                </tr>
```

Comment:
Hi @bradymiller  and @tywrenn 

I found the `<tr class="head">` this `.head` class seems like outdated.
Can I remove it?

Reply:
@stu01509 Since the entire codebase is riddled with this I think it would be a hassle to remove so I would just let it be.

Reply:
That may be but; the goal of modernization is to clean this kind of stuff up, is it not?
If a class is no longer used or an existing custom class can be replaced with a BS class then, should do it while there. Then keep a running list of replaced classes so rest of codebase can be cleaned up. Same with out of place or unused id attributes.

Reply:
@sjpadgett Trueeeee. Let me examine what the class does first and i'll see if it can be replaced.

Reply:
Cool. Teach Cliff what've done to determine if replace is needed or not so he can use in future. Thanks....

Reply:
btw: Debugging Batch Payments while rockin to ZZ TOP.

Reply:
![external-content duckduckgo com](https://user-images.githubusercontent.com/5618372/85886790-b1034800-b7b4-11ea-8fa0-ba42b9279db0.gif)

Reply:
@stu01509 @sjpadgett Turns out -- this code does nothing xD

The reason it does nothing is cause it used to be a color for the table header when that wasn't a thing. Now with BS4 we can color it to whatever we want. I suggest replacing it with something such as `table-primary`.

Reply:
haha, I removed it, and add `table-primary` looks like clearly.

![image](https://user-images.githubusercontent.com/22230889/85910955-6fc27500-b854-11ea-8013-9816c74b26f7.png)